portfolio/src/main/webapp/index.html

```diff
@@ -90,10 +90,31 @@ <h5 class="text-white h4">Menu</h5><br>
     <br><br><br>
   </div>
 
+  <div class="row peach">
+    <div class="col-2">
+    </div>
+    <div class="col-8">
+      <hr>
+    </div>
+  </div>
+
   <div class="row peach">
 	<div class="col center-text">
-      <button class="btn btn-outline-primary" onclick="getHello()">Fetch the hello!</button>
-      <div id="hello-container"></div>
+      <button class="btn btn-outline-primary" onclick="getComments()">Fetch the comments!</button>
```

Comment:
Is this button still needed?  I think most of the comments are loaded when the page opens and `<body onload="getCommentSection()">` is run, right?

Reply:
No, I had just kept it to be able to look at the json, but I will delete it now.

Reply:
Cool.  BTW once you get started using the Chrome debugger you will be able to view the JSON sent from the server without writing any extra JavaScript code